app/views/main_template.scala.html

```diff
@@ -25,10 +26,25 @@
         bodyClasses: Option[String] = None,
         mainClass: Option[String] = None,
         scriptElem: Option[Html] = None,
-        backEnabled: Boolean = true)(mainContent: Html)(implicit request : Request[_], messages: Messages, appConfig: FrontendAppConfig)
+        backEnabled: Boolean = true,
+        timeout: Boolean = true)(mainContent: Html)(implicit request : Request[_], messages: Messages, appConfig: FrontendAppConfig)
 
 @import uk.gov.hmrc.play.views.html.layouts
 
+@scripts = {
+    @if(timeout) {
+        <script id="timeoutScript" role="dialog" tabindex="-1">
+                window.GOVUK.timeoutDialog({
+                    timeout: @appConfig.timeout,
+                    countdown: @appConfig.countdown,
+                    keepAliveUrl: '/register-for-vat/renew-session',
+                    signOutUrl: '/register-for-vat/error/destroy-session'
```

Comment:
This endpoint destroy the session properly, but
1. the content is not correct, and the link doesn't do what the text says it does (look at HMRC Design Patterns)
2. This timeout page creates an infinite loop in the way that, since the script is targeting all microservice pages, after each timeout the timer restarts and there will be another one and so on :point_down:. The timer needs to be disabled on this page.

![Screenshot 2020-11-26 at 17 18 34](https://user-images.githubusercontent.com/6173961/100382288-e980a980-3012-11eb-8e8c-d9a8fc7f47e1.png)